repoman/repository.py

```diff
@@ -377,7 +377,7 @@ def commit(self, message, custom_parent=None, allow_empty=False):
                              with this parameter you can specify a custom one
                              like when there was a fastforward merge and you
                              want to perform a commit anyway
-        :type custom_parent: pygit2.Oid
+        :type custom_parent: DEPRECATED?
```

Comment:
Considering the comment about this parameter, operations to create commits when doing merges would be needed.
Not sure whether or not this parameter is still used in repoman.

Reply:
`custom_parent` parameter is ignored in mercurial implementation, so I guess that if something is using it, then it is git specific and should rather use the no-ff merge strategy for that.
If we still want/need this, I think I can implement it with `git merge -s ours`.

Reply:
I have finally implemented this with `write-tree` and `commit-tree`, I don't like it so much, but tests are passing. I have also added an specific test for custom_parent.